src/common/config/config.cpp

```diff
@@ -305,26 +306,45 @@ void Config::setupDefaultConfig()
 	serverMode = bootBuild ? MODE_CLASSIC : MODE_SUPER;
 	pDefault->strVal = txtServerModes[2 * serverMode];
 
-	pDefault = &defaults[KEY_TEMP_CACHE_LIMIT];
+	defaults[KEY_REMOTE_FILE_OPEN_ABILITY].boolVal = bootBuild;
+
+	//pDefault = &entries[KEY_WIRE_CRYPT].default_value;
+//	if (!*pDefault)
+//		*pDefault == (ConfigValue) (xxx == WC_CLIENT) ? WIRE_CRYPT_ENABLED : WIRE_CRYPT_REQUIRED;
+
+}
+
+void Config::fixDefaults()
+{
+	fb_assert(defaultConfig);
+
+	ConfigValue* pDefault = &defaults[KEY_TEMP_CACHE_LIMIT];
+	ConfigValue* pValue = &values[KEY_TEMP_CACHE_LIMIT];
 	if (pDefault->intVal < 0)
 		pDefault->intVal = (serverMode != MODE_SUPER) ? 8388608 : 67108864;	// bytes
 
-	defaults[KEY_REMOTE_FILE_OPEN_ABILITY].boolVal = bootBuild;
+	if (pValue->intVal < 0)
+		pValue->intVal = pDefault->intVal;
+
```

Comment:
Two blank lines inside a function body looks excessive (per function separation in this file is one line).

Reply:
It allows to visually separate handling of each key.
I can change blank line by additional comment, but will it be better ?